src/dom.js

```diff
@@ -383,3 +383,19 @@ export function getDataParamsFromAttributes(element, opt_computeParamNameFunc) {
   }
   return params;
 }
+
+
+/**
+ * Whether the element has been already fully parsed by the browser.
+ * @param {!Element} element
+ * @return {boolean}
+ */
+export function isElementFullyParsed(element) {
+  let currentElement = element;
+  do {
+    if (currentElement.nextSibling) {
```

Comment:
Using `#nextSibling` also has the benefit of whitespace nodes:

```html
<div>
  <amp-element></amp-element>
</div>
```
There are 2 `TextNodes` in that, and importantly one is is directly after the `amp-element` closing tag. With `#nextSibling`, we may never traverse the parents at all.